include/mpm_explicit.tcc

```diff
@@ -142,6 +142,16 @@ bool mpm::MPMExplicit<Tdim>::solve() {
 
     task_group.wait();
 
+    // Assign material ids to node
```

Comment:
Add an `if` condition for only when there is an interface or at least two active material models?